app/src/test/java/com/mstudio/superheroarch/repository/TheMovieDbRepositoryTest.kt

```diff
@@ -0,0 +1,43 @@
+package com.mstudio.superheroarch.repository
+
+import com.mstudio.superheroarch.BuildConfig
+import com.mstudio.superheroarch.RickAndMortyRepositoryInstruments
+import com.mstudio.superheroarch.remotedatasource.api.TheMovieDbApi
+import kotlinx.coroutines.test.runTest
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mockito.mock
+import org.mockito.Mockito.`when`
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.annotation.Config
+import retrofit2.Response
+
+@Config(manifest = Config.NONE)
+@RunWith(RobolectricTestRunner::class)
+class TheMovieDbRepositoryTest {
+
+    private lateinit var theMovieDbApiMock: TheMovieDbApi
+    private lateinit var theMovieDbRepository: TheMovieDbRepository
+
+    @Before
+    fun before() {
+        theMovieDbApiMock = mock()
+        theMovieDbRepository = TheMovieDbRepository(theMovieDbApiMock)
+    }
+
+    @Test
+    fun `given themoviedb repository, when retrieve episode extra data, then return extra data`() = runTest {
+        val expectedResult = RickAndMortyRepositoryInstruments.givenAnEpisodeExtraData()
+        `when`(theMovieDbApiMock.getRickAndMortyEpisodeDetails("Bearer ${BuildConfig.TMDB_API_KEY}", 1, 1)).thenReturn(Response.success(expectedResult))
+        val result = theMovieDbRepository.getRickAndMortyEpisodeDetails(1, 1)
+        assertEquals(expectedResult.copy("https://image.tmdb.org/t/p/w500image"), result)
```

Comment:
This is weird. You should not have the need to test this conversion here. For me this is an indicator this logic is not in the right place. Why are you forced to do this? Think where else it could be and what changes you could do to avoid it.